Resources/Maps/_NF/Shuttles/gazpacho.yml

```diff
@@ -0,0 +1,5718 @@
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 2.5,1.5
+      parent: 1
+    - type: DeviceLinkSink
+      links:
+      - 345
+  - uid: 575
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -2.5,6.5
+      parent: 1
+    - type: DeviceLinkSink
+      links:
+      - 345
+- proto: SignalButtonDirectional
+  entities:
+  - uid: 221
+    components:
+    - type: Transform
+      pos: 1.5,5.5
+      parent: 1
+    - type: DeviceLinkSource
+      linkedPorts:
+        60:
+        - Pressed: Toggle
+  - uid: 232
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -5.5,-0.5
+      parent: 1
+    - type: SignalSwitch
+      state: True
+    - type: DeviceLinkSource
+      linkedPorts:
+        576:
+        - Pressed: Toggle
+- proto: SignAtmosMinsky
+  entities:
+  - uid: 544
+    components:
+    - type: Transform
+      pos: 1.5,-4.5
+      parent: 1
+- proto: SignBridge
+  entities:
+  - uid: 541
+    components:
+    - type: Transform
+      pos: -4.5,2.5
+      parent: 1
+- proto: SignCanisters
+  entities:
+  - uid: 485
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 5.5,-5.5
+      parent: 1
+- proto: SignElectricalMed
+  entities:
+  - uid: 522
+    components:
+    - type: Transform
+      pos: -0.5,-4.5
+      parent: 1
+- proto: SignEVA
+  entities:
+  - uid: 549
+    components:
+    - type: Transform
+      pos: -5.5,-7.5
+      parent: 1
+- proto: SignFlammableMed
+  entities:
+  - uid: 466
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 4.5,3.5
+      parent: 1
+- proto: SignLastIdiot
+  entities:
+  - uid: 521
+    components:
+    - type: Transform
+      pos: -2.5,-4.5
+      parent: 1
+- proto: SignMedical
+  entities:
+  - uid: 543
+    components:
+    - type: Transform
+      pos: -0.5,2.5
+      parent: 1
+- proto: SignMinerDock
+  entities:
+  - uid: 467
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -6.5,-4.5
+      parent: 1
+  - uid: 468
+    components:
+    - type: Transform
+      pos: 6.5,4.5
+      parent: 1
+- proto: SignSpace
+  entities:
+  - uid: 470
+    components:
+    - type: Transform
+      pos: -3.5,-8.5
+      parent: 1
+  - uid: 471
+    components:
+    - type: Transform
+      pos: -0.5,-7.5
+      parent: 1
+  - uid: 472
+    components:
+    - type: Transform
+      pos: 4.5,4.5
+      parent: 1
+  - uid: 473
+    components:
+    - type: Transform
+      pos: 6.5,-1.5
+      parent: 1
+- proto: SinkWide
+  entities:
+  - uid: 578
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 0.5,4.5
+      parent: 1
+- proto: SMESBasic
+  entities:
+  - uid: 118
+    components:
+    - type: Transform
+      pos: -0.5,-2.5
+      parent: 1
+  - uid: 310
+    components:
+    - type: Transform
+      pos: 1.5,-2.5
+      parent: 1
+- proto: SpawnMobCrabAtmos
```

Comment:
Can be removed, we should try to cut on mobs in ships.

Reply:
isRemoved()